src/tools/opt-dist/src/exec.rs

```diff
@@ -144,13 +144,13 @@ impl Bootstrap {
         self
     }
 
-    pub fn llvm_bolt_instrument(mut self) -> Self {
-        self.cmd = self.cmd.arg("--llvm-bolt-profile-generate");
+    pub fn with_llvm_bolt_ldflags(mut self) -> Self {
+        self.cmd = self.cmd.arg("--set").arg("llvm.ldflags=-Wl,-q");
```

Comment:
will this append a flag or replace the possibly existing `ldflags` in the config ?

Reply:
It appends: https://github.com/rust-lang/rust/blob/3e03a48c18c4465380dca49a41add1609c09d2d8/src/bootstrap/llvm.rs#L716.